internal/cli/terraform.go

```diff
@@ -82,17 +89,38 @@ func generateTerraformCmdRun(cli *cli, inputs *terraformInputs) func(cmd *cobra.
 			return err
 		}
 
-		if err := generateTerraformConfigFiles(inputs.OutputDIR, data); err != nil {
+		if err := generateTerraformImportConfig(inputs.OutputDIR, data); err != nil {
 			return err
 		}
 
-		cli.renderer.Infof("Terraform config files generated successfully.")
+		if terraformProviderCredentialsAreAvailable() {
+			if err := generateTerraformResourceConfig(cmd.Context(), inputs.OutputDIR); err != nil {
+				return err
+			}
+
+			cli.renderer.Infof("Terraform resource config files generated successfully in: %q", inputs.OutputDIR)
+			cli.renderer.Infof(
+				"Review the config and generate the terraform state by running: \n\n	cd %s && ./terraform apply",
```

Comment:
Am I reading this correctly that we install the Terraform binary in the working directory? This seems odd that this would be dangling and no cleanup occurs.

Reply:
Yes, that's correct, the removal call for the binary was removed, this is so users aren't forced to upgrade their terraform version and can use the downloaded binary to generate the plan and apply as terraform 1.5 is needed.

Reply:
Added info about being able to remove the files [`a3bb523` (#796)](https://github.com/auth0/auth0-cli/pull/796/commits/a3bb523dd516fdcc50f7661f38ae614effce8511)